### DISCUSSION TITLE
Error Execution failed for task ':shared:copyResourcesPodDebugFrameworkIosSimulatorArm64'.

### DISCUSSION THREAD

**Discussion by haqim007** [icerockdev/moko-resources#774] [Q&A] [answered]
![Screenshot 2024-09-13 at 16 47 44](https://github.com/user-attachments/assets/83059ab0-6fd7-407e-a649-fb3e31437b33)

Help! Why is it error? 
**Context**
I use KMM as external framework and extract the xcframework then import it via cocoapods 
![Screenshot 2024-09-13 at 16 53 06](https://github.com/user-attachments/assets/87028612-a915-4145-ab1a-9353cce6ce9e)

I follow [this step](https://github.com/icerockdev/moko-resources?tab=readme-ov-file#with-orgjetbrainskotlinnativecocoapods).

**The Error Message when trying to build the ios app**
 

`
FAILURE: Build failed with an exception.  * What went wrong: Execution failed for task ':shared:copyResourcesPodDebugFrameworkIosSimulatorArm64'. > Cannot access output property 'outputDirectory' of task ':shared:copyResourcesPodDebugFrameworkIosSimulatorArm64'. Accessing unreadable inputs or outputs is not supported. Declare the task as untracked by using Task.doNotTrackState(). For more information, please refer to https://docs.gradle.org/8.7/userguide/incremental_build.html#sec:disable-state-tracking in the Gradle documentation.    > java.nio.file.FileSystemException: /Users/xxx/Library/Developer/Xcode/DerivedData/Scrumdinger-fkjvmxzckvbmoiasvmpypevlwthz/Build/Products/Debug-iphonesimulator/Scrumdinger.app: Operation not permitted  * Try: > Run with --stacktrace option to get the stack trace. > Run with --info or --debug option to get more log output. > Run with --scan to get full insights. > Get more help at https://help.gradle.org.  BUILD FAILED in 1s 15 actionable tasks: 3 executed, 12 up-to-date Configuration cache entry reused.
`

**Comment by haqim007** [comment]
please help @Alex009

**Comment by Alex009** [comment, marked as answer]
@haqim007 you should disable sandbox of build scripts - https://github.com/icerockdev/moko-resources?tab=readme-ov-file#xcode-setup

**Reply by haqim007** [reply]
I tried to clean and build again and it works.
